src/pages/dashboard/_StateCumulativeTestsContainer.js

```diff
@@ -127,6 +128,16 @@ function groupAndSortStateDaily(query) {
   }
 }
 
+// TODO: we're iterating over data and calling parseDate multiple times in this
+// component. Seems like it could be optimized.
```

Comment:
I agree, this could be optimized. We could do some of this work before we send it off to `groupAndSortStateDaily()` since the data that comes back from the query is actually all the data and we can just run `extent` with that. But I think its fine for now, especially since its only going to happen at build time.

Reply:
Yup, so I started trying to do that, but ran into all kinds of headaches. There's `propTypes` to contend with in `AreaChart`, and the sorting function, and just a bunch of stuff. I figured this will do for V1 launch, and we can refactor later, prioritizing against other improvements.